### PR TITLE
Create a README for XCMv2 detailing notable changes

### PR DESCRIPTION
**Pull request by KiChjang**

### CODE REVIEW

**Comment by apopiak** [comment]
btw. if the content is the same you could potentially just use https://github.com/livioribeiro/cargo-readme to generate the readme.md

**Review by shawntabrizi — APPROVED** [review]

File: xcm/src/v1/README.md [hunk 1]
```diff
@@ -1,25 +0,0 @@
-# XCM Version 1
-The comprehensive list of changes can be found in [this PR description](https://github.com/paritytech/polkadot/pull/2815#issue-608567900).
-
-## Changes to be aware of
-Most changes should automatically be resolved via the conversion traits (i.e. `TryFrom` and `From`). The list here is mostly for incompatible changes that result in an `Err(())` when attempting to convert XCM objects from v0.
-
-### Junction
```
Comment (shawntabrizi) [inline xcm/src/v1/README.md:7]: why remove this file?

Comment (KiChjang) [reply xcm/src/v1/README.md:7]: There was a discussion where we agreed that we shouldn't maintain duplicated information on 2 different locations (i.e. both on the README.md and the top-level doc comments for the Rust module), and so I opted for only writing it on the top-level doc comments.

**Comment by KiChjang** [comment]
bot merge

**Pull request merged**